src/test/java/ru/testtask/service/ProjectServiceTest.java

```diff
@@ -1,43 +1,54 @@
+package ru.testtask.service;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
+import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.junit4.SpringRunner;
+import ru.testtask.Application;
 import ru.testtask.exception.NameAlreadyExistsException;
+import ru.testtask.exception.WrongMethodUseException;
 import ru.testtask.model.Attribute;
 import ru.testtask.model.Geometry;
 import ru.testtask.model.Project;
 import ru.testtask.repo.ProjectRepo;
 import ru.testtask.service.ProjectService;
+import ru.testtask.service.UserService;
+
 import java.util.List;
 
 import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
 
 
 @RunWith(SpringRunner.class)
+@SpringBootTest(classes = {Application.class})
 public class ProjectServiceTest {
```

Comment:
Please add new test method to check `ProjectService#isCurrentUserOwnerOf(String id)`